quarkchain/cluster/miner.py

```diff
@@ -432,15 +432,14 @@ def _log_status(block: Block):
         full_shard_id = "R" if is_root else block.header.branch.get_full_shard_id()
         count = len(block.minor_block_header_list) if is_root else len(block.tx_list)
         elapsed = time.time() - block.header.create_time
-        Logger.info_every_sec(
+        Logger.info(
             "[{}] {} [{}] ({:.2f}) {}".format(
                 full_shard_id,
                 block.header.height,
                 count,
                 elapsed,
                 block.header.get_hash().hex(),
             ),
-            60,
         )
```

Comment:
we have to set 60 back here since we have already log root block in AddRootBlock request

Reply:
good point

Reply:
Got it~